services/QuillLMS/client/app/bundles/Teacher/components/assignment_flow/create_unit/assign_a_new_activity.tsx

```diff
@@ -91,16 +115,54 @@ const AssignANewActivity = ({ numberOfActivitiesAssigned, showDiagnosticBanner }
     </section>)
   }
 
+  const suggestedActivitiesList = activitiesToSuggest.length > 0 &&
+  (
+    <div>
+      <h1 className="evidence-header">Browse the newest Reading for Evidence activities</h1>
+      <table className="data-table suggested-activities-table">
+        <tr className="data-table-headers">
+          <th className="data-table-header">Tool</th>
+          <th className="data-table-header activity-header">Activity</th>
+          <th className="data-table-header topics-header">Topics</th>
+          <th className="data-table-header">Date released</th>
+        </tr>
+        <tbody className="data-table-body">
+          <div className="list-item">
+            {activitiesToSuggest.map((a) => {
+              return (
+                <tr className="data-table-row" key={a.id}>
+                  <span className="tool-col"><img alt={evidenceToolIcon.alt} src={evidenceToolIcon.src} /></span>
+                  <td className="data-table-row-section name-col">
+                    <a href={`/assign/activity-library?activityClassificationFilters[]=evidence&search=${encodeURI(a.name)}`}>{a.name}</a>
+                  </td>
+                  <td className="data-table-row-section topics-col">
+                    {a.topics.map((t, i) => {
+                      return <p key={i}>{t.join(" / ")}</p>
+                    })}
+                  </td>
+                  <td className="date-col">{a.publication_date}</td>
+                  <td className="preview-col"><a href={`/activity_sessions/anonymous?activity_id=${a.id}`} rel="noopener noreferrer" target="_blank">Preview</a></td>
+                  <td className="select-col"><button className="quill-button secondary medium focus-on-light outlined select-suggested" onClick={() => window.location.href = `/assign/activity-library?activityClassificationFilters[]=evidence&search=${encodeURI(a.name)}`} type="button">Select</button></td>
+                </tr>
+              )
+            })}
+          </div>
+        </tbody>
+      </table>
```

Comment:
Is there a reason you aren't using the `<DataTable />` Backpack component here? I think using the styling from that component but not the component itself risks making a change to that CSS at some point and breaking this instance, since the HTML won't necessarily remain consistent.